cmd/descheduler/app/version.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+// This becomes easy for exporting later. In fact, this could be moved to top level directory.
+const DeschedulerVersion = "0.1.0"
```

Comment:
Instead of hard coding, could you check how to print something like, for example, last tag+commits since last tag or something similar. You could also see how it is done in kube/openshift, and have something similar.

Reply:
Sure, I initially wanted to do that, looking at the kube code but thought it was unnecessary at this stage but I can.

Reply:
actually hard coding 0.1.0 might give wrong information even at this stage, and also it would require updating whenever we have a new release with a new tag.